include/crm_internal.h

```diff
@@ -91,6 +91,7 @@
 #define PCMK__XA_DC_LEAVING             "dc-leaving"
 #define PCMK__XA_DIGEST                 "digest"
 #define PCMK__XA_ELECTION_AGE_SEC       "election-age-sec"
+#define PCMK__XA_ELECTION_AGE_USEC      "election-age-nano-sec"
```

Comment:
Will do. I thought this one was just too bad haha.

Reply:
Need to rebase on current main too

Reply:
Couple of questions.
1. There's an internal constant I named PCMK__META_CLONE_INSTANCE_NUM that I named because PCMK__META_CLONE seemed too unclear at the time. That's already merged. Shall I rename it?
2. There are, rarely, attributes that are too long to specify exactly in the constant name. A much later commit has PCMK_NODE_ATTR_RSC_DISCOVERY_EENABLED because PCMK_NODE_ATTR_RESOURCE_DISCOVERY_ENABLED is ridiculously long and doesn't even fit within the alignment. There might be one other such case. In those cases, should we abbreviate slightly or use the full literal in the name?

Reply:
> Couple of questions.
> 
>     1. There's an internal constant I named PCMK__META_CLONE_INSTANCE_NUM that I named because PCMK__META_CLONE seemed too unclear at the time. That's already merged. Shall I rename it?

Ah, yes, I missed that one. 
 
>     2. There are, rarely, attributes that are too long to specify exactly in the constant name. A much later commit has PCMK_NODE_ATTR_RSC_DISCOVERY_ENABLED because PCMK_NODE_ATTR_RESOURCE_DISCOVERY_ENABLED is ridiculously long and doesn't even fit within the alignment. There might be one other such case. In those cases, should we abbreviate slightly or use the full literal in the name?

I'm thinking we should use the full name in the headers, but it might make sense for callers to `#define` a short alias if it's too awkward using the full name. FYI I want to deprecate that particular constant, since a location constraint with `rsc-pattern=".*" resource-discovery="never"` is equivalent and more consistent.

Reply:
> Ah, yes, I missed that one.

Adding a commit on top.

> I'm thinking we should use the full name in the headers

I updated the later commit (not in this PR) to do so. There might be one other like I said, so just let me know if you see one during review.